pkg/apiclient/argo-server-opts.go

```diff
@@ -17,6 +17,10 @@ type ArgoServerOpts struct {
 	// use custom http client
 	HTTP1Client *http.Client
 	Headers     []string
+
+	// client Certificates
```

Comment:
```suggestion
	// client certificate
```
singular, plus use consistent spacing and capitalization as the other opts